pages/profile-setup/components/ProfileSetupForm/ProfileSetupForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { useRouter } from "next/router";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { AxiosError } from "axios";
+import classNames from "classnames/bind";
+import { useForm } from "react-hook-form";
+
+import { postProfileSetup } from "@/apis/apis";
+import Input from "@/components/Input";
+import { PAGE_PATH } from "@/constants/pagePath";
+
+import FormSchema from "./FormSchema";
+import styles from "./ProfileSetupForm.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ProfileSetupFormProps {
+  accountId: number;
+  email: string;
+}
+
+export interface FieldValuesType {
+  nickname: string;
+  phoneNumber: string | null;
+}
+
+export default function ProfileSetupForm({ accountId, email }: ProfileSetupFormProps) {
+  const router = useRouter();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FieldValuesType>({
+    mode: "all",
+    defaultValues: { nickname: "", phoneNumber: null },
+    resolver: zodResolver(FormSchema),
+  });
+
+  const onSubmit = async (data: Omit<FieldValuesType, "accountId">) => {
```

Comment:
```suggestion
  const onSubmit = async (data: Omit<FieldValuesType, "accountId">) => {
```
p4: 여기서 `data` 타입은 `FieldValuesType`을 적용해도 될 것 같네요...?
`FieldValuesType`에 `accountId `가 없는 것 같은데 `Omit`으로 제외시키고 있는 것 같아요!

Reply:
헉 세상에 이렇게 멍청할수가...? 왜저랬을까여...?